src/main/java/io/botcrafting/botcraft/core/handler/processors/AnswerBotcraftMention.java

```diff
@@ -0,0 +1,35 @@
+package io.botcrafting.botcraft.core.handler.processors;
+
+import static io.botcrafting.botcraft.configuration.constant.MessageConstant.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import io.botcrafting.botcraft.core.handler.chain.MessageChain;
+import io.botcrafting.botcraft.core.model.Message;
+import io.botcrafting.botcraft.core.service.MessageSenderService;
+
+@Component
+public class AnswerBotcraftMention implements MessageProcessor{
```

Comment:
For all classes that implements Message Processor maybe they would need a suffix if makes sense. Like BotcraftMentionReplier